### PR TITLE
fix(ui): ui bug fix for datasets sidebar stats section

### PR DESCRIPTION
**Pull request by ShubhamThakre**
## Checklist
- Datasets sidebar Stats section vertical height fix
Screenshot 2022-06-01 at 1 21 33 AM
Screenshot 2022-06-01 at 1 21 55 AM

@jjoyce0510 @chriscollins3456

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx [hunk 1]
```diff
@@ -60,6 +60,10 @@ export const SidebarStatsSection = () => {
 
     const lastUpdatedTime = latestOperation && toLocalDateTimeString(latestOperation?.lastUpdatedTimestamp);
 
+    const hasUsageStatsAggregations =
+        usageStats?.aggregations?.totalSqlQueries || (usageStats?.aggregations?.users?.length || 0) > 0;
+    const hasLatestProfiles = latestProfile?.rowCount || latestProfile?.columnCount;
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx:65]: is this used?

File: datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx [hunk 2]
```diff
@@ -71,51 +75,55 @@ export const SidebarStatsSection = () => {
...
+                        <InfoItem title="Top Users" width={INFO_ITEM_WIDTH_PX}>
+                            <UsageFacepile users={usageStats?.aggregations?.users} maxNumberDisplayed={10} />
+                        </InfoItem>
+                    ) : null}
+                </StatsRow>
+            )}
             {/* Operation Entry */}
-            <StatsRow>
-                {latestOperation?.timestampMillis ? (
+            {latestOperation?.timestampMillis ? (
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx:116]: should we call this hasLatestOperation?

Comment (ShubhamThakre) [reply datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Dataset/StatsSidebarSection.tsx:116]: Sure, I update the code and push the changes.

**Review by jjoyce0510 — APPROVED** [review]
Thank you!

**Pull request merged**